webapp/store/views.py

```diff
@@ -131,27 +138,20 @@ def user_details(username):
 @jaasstore.route(
     "/u/<username>/<charm_or_bundle_name>/<series_or_version>/<version>"
 )
-def user_entity(
-    username, charm_or_bundle_name, series_or_version=None, version=None
-):
+def user_entity(username, charm_or_bundle_name):
     charmhub_url = (
         "https://charmhub.io/" + username + "-" + charm_or_bundle_name
     )
-    return details(
-        charm_or_bundle_name,
-        series_or_version=series_or_version,
-        version=version,
-        charmhub_url=charmhub_url,
-    )
```

Comment:
**Not a blocker** but why not keep this in here and let the details function handle the redirect?